android/src/main/java/com/stripeterminalreactnative/ktx/TerminalExtensions.kt

```diff
@@ -131,10 +121,12 @@ suspend fun Terminal.connectReader(
             connectHandoffReader(reader, HandoffConnectionConfiguration(locationId), listener)
         else connectHandoffReader(reader, HandoffConnectionConfiguration(locationId))
     }
-    EMBEDDED -> connectEmbeddedReader(reader, EmbeddedConnectionConfiguration(Unit))
     USB -> {
         if (listener is UsbReaderListener)
             connectUsbReader(reader, UsbConnectionConfiguration(locationId), listener)
         else connectUsbReader(reader, UsbConnectionConfiguration(locationId))
     }
+    else -> {
+        throw IllegalArgumentException("Unsupported discovery method: ${discoveryMethod}")
```

Comment:
EMBEDDED is still present in types, but according to Android release notes it's not for public use